swagger/communication/communication.json

```diff
@@ -0,0 +1,1064 @@
+{
```

Comment:
The response formats are not consistent.
When you get a list, it should be an array under a property with the name of the entity returned (plural), when you return a single entity it should be an object directly under the data property.
Its easier to see here:
https://intandem-internal.readme.io/reference/settingscontroller_create

voicecallsetting 
post/get/update should return a single object with no "setting" property

voice call
get list should return a list of entities, not a string (the rest are returning good responses)

Quota
Change the tag to "Voice Call Quota"
return object should be directly under data (no "quota" property

Reply:
In regards to the get settings - let's assume I am a consumer and I only have a business uid, and I wish to get that business' settings (I don't have a settingsuid).
Is this supposed to be done with a get settings list call?

Reply:
And in regards to the changes to 'quotas', which APIs are you referring to?

Reply:
I think goldi knows